src/modules/emote_menu/utils/twitch-emotes.js

```diff
@@ -113,6 +170,20 @@ export async function loadTwitchEmotes() {
         newToken = emoteToken;
       }
 
+      let predicate = () => false;
+      if (['SUBSCRIPTIONS', 'FOLLOWER'].includes(type)) {
+        // eslint-disable-next-line react/function-component-definition
+        predicate = () => {
```

Comment:
rather than defining a new method for each emote in memory, we could write a single predicate that works for all of them, and then pass in an emote id, right?